pkg/console/dashboard_panels.go

```diff
@@ -330,6 +313,17 @@ func chartIsInstalled() bool {
 	return processed >= 1
 }
 
+func isAPIReady(managementURL, path string) bool {
+	if !strings.HasPrefix(current.managementURL, "https://") {
+		return false
+	}
+	command := fmt.Sprintf(`curl -k %s%s`, managementURL, path)
+	cmd := exec.Command("/bin/sh", "-c", command)
+	cmd.Env = os.Environ()
+	_, err := cmd.CombinedOutput()
+	return err == nil
```

Comment:
Do we need to check HTTP status here? Or a response is sufficient?

Reply:
changed to 'curl -fk'